src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1958,7 +1958,7 @@
 
 helps['tag'] = """
 type: group
-short-summary: Manage resource tags.
+short-summary: Tags Management on a resource.
```

Comment:
Please write help with the first person voice
```suggestion
short-summary: Tag Management on a resource.
```